esp32-hal/src/lib.rs

```diff
@@ -34,6 +34,11 @@ pub mod analog {
     pub use esp_hal_common::analog::{AvailableAnalog, SensExt};
 }
 
+/// Serial Configuration
+pub mod serial {
+    pub use esp_hal_common::serial::*;
+}
```

Comment:
Is there a reason we can't just do `pub use esp_hal_common::serial;` here, like we do with the other driver modules?

(Same for the 3 remaining chips)

Reply:
I think for the docs ... but they look okay even with just the usual `pub use`